### ISSUE TITLE
Default reporters can't be turned off anymore

### ISSUE THREAD

**Issue by analog-nico** [SBoudrias/gulp-istanbul#102]
Since changing `_.defaults` to `_defaultsDeep` in commit https://github.com/SBoudrias/gulp-istanbul/commit/e94f537aee12a422bb0ad816fb42f7e5de59d1fc the default reporters are always turned on.

I just use `reporters: ['lcov']` and now all other default reporters like the report in the console are executed as well.

According to the scope of #79 it would be sufficient to just do the `_.defaultsDeep` on `reportOpts`:

``` diff
- opts = _.defaultsDeep(opts, {
+ opts = _.defaults(opts, {
    coverageVariable: COVERAGE_VARIABLE,
    dir: defaultDir,
    reporters: [ 'lcov', 'json', 'text', 'text-summary' ]
+ });
+ opts = _.defaultsDeep(opts, {
    reportOpts: {
      dir: opts.dir || defaultDir
    }
  });
```

**Comment by ecozoic** [comment]
Would this issue be what's causing the text-summary reporter to run even when I'm only specifying json?

**Comment by analog-nico** [comment]
@ecozoic Exactly!

**Comment by SBoudrias** [comment]
Fixed with v1.1.1

**Comment by analog-nico** [comment]
Works nicely. Thanks!
